torchtitan/config_manager.py

```diff
@@ -480,6 +480,20 @@ def __init__(self):
             help="Flight recorder ring buffer size, >0 means recording by default, 0 means disabled",
         )
 
+        # estimation mode settings
+        self.parser.add_argument(
+            "--estimate.memory",
```

Comment:
Do we estimate anything other than memory? If not, we should name it to `--memory_estimation.enabled`

Reply:
We plan to extend the `estimation.py` to include run-time/MFU estimation. In that case we will use 
`--estimate.mfu=True` or `--estimate.runtime=True`

Reply:
Oh that makes sense. Can they be run simultaneously? If so, I think it's better to have a global switch, e.g. `--estimate.enable` or something to put in `run_llama_train.sh`